.cm/gitstream.cm

```diff
@@ -0,0 +1,77 @@
+# -*- mode: yaml -*-
+# This example configuration for provides basic automations to get started with gitStream.
+# View the gitStream quickstart for more examples: https://docs.gitstream.cm/examples/
+manifest:
+  version: 1.0
+
+
+triggers:
+  exclude:
+    branch:
+      - l10n_dev
+      - dev
```

Comment:
@coderabbitai count total of all merged PRs for milestone 1.19.0 that has a bug label.

Reply:
That's incorrect, should be 51

Reply:
Redo the search but set a limit of 100 results